auth-service/config/debug.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+
+    "include" => env("DEBUG_MODE", false),
```

Comment:
добавить приведение к типу
```suggestion
    "include" => (bool) env("DEBUG_MODE", false),
```